modules/wpcom-block-editor/class-jetpack-wpcom-block-editor.php

```diff
@@ -86,6 +86,25 @@ public function add_iframed_body_class( $classes ) {
 		return $classes;
 	}
 
+	/**
+	 * Checks to see if cookie can be set in current context. If 3rd party cookie blocking
+	 * is enabled the editor can't load in iFrame, so emiting X-Frame-Options: DENY will
+	 * force the editor to break out of the iFrame.
+	 */
+	private function check_iframe_cookie_setting() {
+		if ( ! isset( $_SERVER['QUERY_STRING'] ) || ! strpos( $_SERVER['QUERY_STRING'], 'calypsoify%3D1%26block-editor' ) || isset( $_COOKIE['wordpress_test_cookie'] ) ) {
+			return;
+		}
+
+		if ( ! $_GET['calypsoify_cookie_check'] ) {
```

Comment:
Could you add a comment to satisfy the linter?

```suggestion
		if ( ! $_GET['calypsoify_cookie_check'] ) { // phpcs:ignore WordPress.Security.NonceVerification.Recommended
```

Reply:
done